react/components/Tooltip/hooks.ts

```diff
@@ -17,7 +17,7 @@ export function useRect(nodeRef, observe = true) {
       observerRef.current.observe()
     }
     return () => observerRef.current && observerRef.current.unobserve()
-  }, [observe, nodeRef.current])
+  }, [observe, nodeRef])
```

Comment:
No problem here in deleting this item?

Reply:
I don't think so, this change was autofixed by the `react-hooks/exhaustive-deps` rule